src/Estina/SmartFocus/Api/Rest/Notification.php

```diff
@@ -79,32 +79,41 @@ public function post(SimpleXMLElement $xmlObject)
     /**
```

Comment:
You probably want to open this file and have a look at the post method + buildTransactionalRequestObject.